recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -15,32 +15,41 @@ def build(self):
         cmake.configure()
         cmake.build()
 
+    @property
+    def _build_context(self):
```

Comment:
Is this used in the recipe?

Reply:
It was 🤦 ... until I realize that `file` was a working command for all the platforms... I'll remove the method.